arelle/plugin/validate/DBA/rules/__init__.py

```diff
@@ -308,3 +308,15 @@ def minimumRequiredFactsFound(
         if count >= requiredCount:
             return True
     return False
+
+
+def consolidated_dimension_exists(modelXbrl: ModelXbrl, consolidatedSoloQn: QName) -> bool:
```

Comment:
Arbitrary line.  The functions and variables you added should be camel case to be consistent with the rest of Arelle.